Website/src/rendering/src/components/Products/ProductDetailsContent.tsx

```diff
@@ -148,14 +150,34 @@ const ProductDetailsContent = ({
     async (e: FormEvent) => {
       e.preventDefault();
       setLoading(true);
-      await dispatch(
+      const response = await dispatch(
         createLineItem({
           ProductID: product.ID,
           Quantity: quantity,
           Specs: specValues,
         })
       );
       setLoading(false);
+
+      const resPayload: { LineItems?: LineItem[] } = response?.payload;
```

Comment:
Is the response containing all the cart line items?
What happens when I have already 2 line items in my cart, I go to the first product PDP, and I add another one of it to my cart? Will the last line item be the new product I added to my cart or the second product which I have not modified the quantity? Would that be a problem in the code here?